health_check/views.py

```diff
@@ -92,16 +92,21 @@ def get(self, request, *args, **kwargs):
         accept_header = request.META.get('HTTP_ACCEPT', '*/*')
         for media in MediaType.parse_header(accept_header):
             if media.mime_type in ('text/html', 'application/xhtml+xml', 'text/*', '*/*'):
-                context = {'plugins': self.plugins, 'status_code': status_code}
+                context = self.get_context_data(**kwargs)
                 return self.render_to_response(context, status=status_code)
             elif media.mime_type in ('application/json', 'application/*'):
                 return self.render_to_response_json(self.plugins, status_code)
         return HttpResponse(
             'Not Acceptable: Supported content types: text/html, application/json',
             status=406,
-            content_type='text/plain'
+            content_type='text/plain',
         )
 
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        context_data['plugins'] = self.plugins
+        return context_data
```

Comment:
I don't know in which Python version this way dictionary unpacking was introduced, but you could write this as:
```suggestion
    def get_context_data(self, **kwargs):
        return {**super().get_context_data(**kwargs), "plugins": self.plugins}
```
This is up to your prefrence though.

Reply:
I also prefer this version with dictionary unpacking :+1: 
This feature was introduced in python 3.5, so I'm going to use it, because in travis the lowest Python version is 3.6